composer.json

```diff
@@ -31,14 +31,14 @@
         "ezsystems/ezplatform-search": "^1.2@dev",
         "ezsystems/ezplatform-user": "^2.3@dev",
         "friendsofphp/php-cs-fixer": "^3.0",
-        "ibexa/doctrine-schema": "~4.2.0@dev",
-        "ibexa/content-forms": "~4.2.0@dev",
-        "ibexa/design-engine": "~4.2.0@dev",
-        "ibexa/user": "~4.2.0@dev",
-        "ibexa/fieldtype-richtext": "~4.2.0@dev",
-        "ibexa/rest": "~4.2.0@dev",
-        "ibexa/search": "~4.2.0@dev",
-        "ibexa/http-cache": "~4.2.0@dev",
+        "ibexa/doctrine-schema": "~4.3.0@dev",
+        "ibexa/content-forms": "~4.3.0@dev",
+        "ibexa/design-engine": "~4.3.0@dev",
+        "ibexa/user": "~4.3.0@dev",
+        "ibexa/fieldtype-richtext": "~4.3.0@dev",
+        "ibexa/rest": "~4.3.0@dev",
+        "ibexa/search": "~4.3.0@dev",
+        "ibexa/http-cache": "~4.3.0@dev",
```

Comment:
Seems like this rebase didn't go so well. I feel responsible for making the fuss :sweat_smile: